integration/maptest/map_test.go

```diff
@@ -212,83 +215,76 @@ func TestInclusion(t *testing.T) {
 	ctx := context.Background()
 	for _, tc := range []struct {
 		desc         string
-		HashStrategy trillian.HashStrategy
+		HashStrategy []trillian.HashStrategy
 		leaves       []*trillian.MapLeaf
 	}{
 		{
-			desc:         "maphasher single",
-			HashStrategy: trillian.HashStrategy_TEST_MAP_HASHER,
+			desc:         "single",
+			HashStrategy: []trillian.HashStrategy{trillian.HashStrategy_TEST_MAP_HASHER, trillian.HashStrategy_CONIKS_SHA512_256},
 			leaves: []*trillian.MapLeaf{
 				{Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("A")},
 			},
 		},
 		{
-			desc:         "maphasher multi",
-			HashStrategy: trillian.HashStrategy_TEST_MAP_HASHER,
+			desc:         "multi",
+			HashStrategy: []trillian.HashStrategy{trillian.HashStrategy_TEST_MAP_HASHER, trillian.HashStrategy_CONIKS_SHA512_256},
```

Comment:
Do you think these tests are enough to show verification works correctly at multiple levels and leaves? It's hard for me to be sure.